src/modeles/autorisations/autorisationBase.js

```diff
@@ -32,6 +33,32 @@ class AutorisationBase extends Base {
     );
   }
 
+  peutGererContributeurs() {
+    return (
+      this.permissionAjoutContributeur && this.permissionSuppressionContributeur
+    );
+  }
+
+  statutDeDroit() {
+    const toutesRubriques = Object.keys(Rubriques);
+
+    const verifieNiveauSurToutesRubriques = (niveau) =>
+      toutesRubriques.every((rubrique) => this.aLaPermission(niveau, rubrique));
```

Comment:
Il ne faut pas utiliser `aLaPermission` ici car :
si un utilisateur a 'LECTURE' partout et un 'ECRITURE' il sera considéré comme 'LECTURE' alors qu'il devrait être mixte

Reply:
Suggestion : compter le nombre d'occurence de chaque niveau. Puis comparer ces nombres avec le nombre de clés de `this.droits` (pour avoir le nombre total de rubriques).